npc/quests/quests_ein.txt

```diff
@@ -4787,7 +3308,7 @@ ein_in01,231,163,7	script	Sick Old Man#ein	4_M_DIEMAN,{
 				mes "^3355FFIt doesn't seem";
 				mes "like this old man";
 				mes "is telling the truth...^000000";
-				einbech = 0;
+				.@einbech = 0;
 				next;
 			}
 			else {
```

Comment:
This is outside of the Uwe Kleine NPC, I was only nice enough to correct typos, won't fix the code-style of the whole file in this PR. The whole file has a shitty code-style.